docs/user/cli/nailgun.rst

```diff
@@ -0,0 +1,31 @@
+.. _nailgun_server:
+
+Nailgun Server
+==============
+
+GeoMesa can use a persistent server to eliminate the overhead of bootstrapping a JVM for each command, which can
+make commands much faster to start. The server must be enabled by setting the environment variable
+``GEOMESA_NG_ENABLED=true``.
+
+.. warning::
+
+  The Nailgun server provides no security, and should only be used in a trusted environment. By default
+  it will bind to all host addresses, allowing unauthenticated connections from any client on the same network.
```

Comment:
This is great.  If I were ever going to push for a Docker, this would be a great reason to make one:).